Composer/packages/electron-server/src/main.ts

```diff
@@ -200,13 +205,30 @@ async function run() {
 
   app.on('ready', async () => {
     log('App ready');
+    const getMainWindow = () => ElectronWindow.getInstance().browserWindow;
+    const { startApp, updateStatus } = await initSplashScreen({
+      getMainWindow,
+      color: 'rgb(0, 120, 212)',
+      logo: `file://${microsoftLogoPath}`,
+      productName: 'Bot Framework Composer',
+      productFamily: 'Microsoft Azure',
+      status: 'Initializing...',
+      website: 'www.botframework.com',
+      width: 500,
+      height: 300,
+    });
+
+    updateStatus('Starting server...');
```

Comment:
Messages like this should be localized, since the app menu was not localized, I assumed the effort is still going on, please feel free to let me know if things are in place for electron localization and how can I use it.

Reply:
@beyackle can you comment?

Reply:
Looks like we're not localizing anything in the `electron-server` directory yet, but we ought to be, with our standard `formatMessage` calls. Let's open another issue for that and handle this string along with all the others.

Reply:
@beyackle sounds good, thanks

Reply:
As a side note, the l10n script does look in `electron-server` already, so no change is needed there. Just get `formatMessage` around all your strings and you'll be good to go.

Reply:
#4120 Here's the issue.